Sources/BraintreePayPal/RecurringBillingMetadata/BTPayPalRecurringBillingDetails.swift

```diff
@@ -40,7 +44,9 @@ public struct BTPayPalRecurringBillingDetails {
         oneTimeFeeAmount: String? = nil,
         shippingAmount: String? = nil,
         productAmount: String? = nil,
-        taxAmount: String? = nil
+        taxAmount: String? = nil,
+        unitAmount: String? = nil,
+        amountBreakdown: BTRecurringBillingAmountBreakdown? = nil
```

Comment:
`amountBreakdown` should now be a parameter on the `BTPayPalCheckoutRequest` instead of `BTPayPalRecurringBillingDetails`